go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aiven/terraform-provider-aiven
 
-go 1.21.1
+go 1.21
```

Comment:
this will always use the latest PATCH version available, which contains only vulnerability and bug fixes